docs/UserGuide.md

```diff
@@ -202,48 +202,61 @@ Example:
 
 ### User Profile
 
-#### Adding to user profile: `add -me`
+Note: You must be viewing the **Profile** page in order to execute these commands.
```

Comment:
Would it be weird if it is "viewing" the profile page? To be cohesive to our DG, `You must be on the **profile** page ...` sounds more cohesive to our precondition